CONTRIBUTING.md

```diff
@@ -49,5 +49,6 @@ In order to maintain a high software quality standard, we strongly prefer contri
 ## Coding Conventions
 
 - Use `println()` only in CLI modules and `log` in library modules.
+- Use `2017-${current-year}` as the date range for copyright statements.  
```

Comment:
I'm not sure we should formulate this as a general convention. What about Copyrights from other contributing parties like Bosch?

Reply:
From our discussion there was `simplicity` as argument provided - which is the reason I've put it here. If that could be a problem for contributers then we should of course not add it here.